libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardScenarioTigerCharts.tsx

```diff
@@ -0,0 +1,9 @@
+// (C) 2023 GoodData Corporation
+import React from "react";
+import { Dashboard } from "@gooddata/sdk-ui-dashboard";
+import { idRef } from "@gooddata/sdk-model";
+import { Dashboards } from "../../../../../reference_workspace/workspace_objects/goodsales/current_reference_workspace_objects_tiger";
+
+export const DashboardScenarioTigerCharts: React.FC = () => {
```

Comment:
just naming, Scenario word should be last

Reply:
Kept in sync with other tiger-specific scenarios.